### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by hamzaw9**
In this pull request:

- Use media queries to make sure website displays correctly on desktop screens.
- Use Grid and Flexbox

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -381,4 +385,207 @@ fieldset textarea {
...
+  }
+
+  .about-left {
+    flex-basis: 41%;
+  }
+
+  .about-right {
+    flex-basis: 59%;
+  }
+
```
Comment (jerryowusu) [inline style.css]: Nice work so far. However, kindly make sure that the left and right sides of the about section maintains have the same height just as in the figma design at all times, and at all screen sizes of the desktop version. The left side seems to be longer than the right side for screen widths below 950px as shown below :point_down: Kindly check and update accordingly. :pray: 
Kindly also consider adjusting the widths to 40% for left side and 60% to right side to match the design. 

### Your design 

Screenshot from 2023-07-12 12-06-36



### Actual design 


Screenshot from 2023-07-12 12-05-45

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @hamzaw9 , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so-close


## Highlights
Nice design :+1: 
Neat and well organized code :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply style.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @hamzaw9 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**